core/crypto/vrf/vrf_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vrf
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestUniqueID(t *testing.T) {
```

Comment:
Correct, and I like that we have such a test! However, this test doesn't test the functionality of UniqueID function, i.e., given an input, test that it generates the expected output. I remember in the tree verifiers test, when we changed the verification algorithm, specifically the way we hash, I had to manually recalculate the hashes and change their values in the test. This was a clear sign to reviewers that a major (and potentially dangerous) change is happening. What do you think?